cmd/playground/window_model.go

```diff
@@ -0,0 +1,207 @@
+package playground
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+
+	"github.com/odpf/optimus/job"
+)
+
+// model contains a cursor to keep track of the pointer, two input fields for taking Size and Offset as input
+// model also contains a state which maintains both versions of windows and the sechdule time
+type model struct {
+	cursor      cursorPointer
+	state       state
+	sizeInput   textinput.Model
+	offsetInput textinput.Model
+}
+
+func initModel() *model {
+	windowV1 := job.WindowV1{Size: defaultSize, Offset: defaultSize, TruncateTo: defaultTruncate}
+	windowV2 := job.WindowV2{Size: defaultSize, Offset: defaultSize, TruncateTo: defaultTruncate}
+	sizeInput := textinput.New()
+	sizeInput.Placeholder = defaultSize
+	sizeInput.Focus()
+	offsetInput := textinput.New()
+	offsetInput.Placeholder = defaultSize
+	return &model{
+		cursor:      pointToSize,
+		state:       state{windowV1: windowV1, windowV2: windowV2, scheduledTime: time.Now()},
+		sizeInput:   sizeInput,
+		offsetInput: offsetInput,
+	}
+}
+
+// since we don not intend to keep any input after the process is killed we return nil
+func (*model) Init() tea.Cmd {
+	return nil
+}
+
+// this handles motion of the pointer when we hit arrow-up key
+func (m *model) handleUp() {
+	switch m.cursor {
+	case pointToOffset:
+		// here we are shifting from Offsetinput to Sizeinput so we have to stop updating the Offsetinput(hence Offset.Blur())
```

Comment:
please update the letter case, for example changing `Offsetinput` to `offsetInput` or `offset input` or anything else, as long as it is consistent with the message to the user or with the variables in the implementation

Reply:
this is still not updated, why already resolved?